CHANGELOG.md

```diff
@@ -8,17 +8,21 @@
 
 ### Minor
 
+- Add idealDirection to Tooltip and IconWithTooltip (#697)
```

Comment:
@ZaphyrRobin Can we add it only to `Tooltip`, `IconWithTooltip` is deprecated since we'll add link to `Tooltip` in #684

Also, change the description to `Tooltip: Add idealDirection`

Reply:
sure, thanks!